### PR TITLE
Fix handling of explorer url

### LINKED ISSUE #1310: [PoolDetail] For chaosnet tx link refers to testnet (instead of chaosnet)
### Change:
https://testnet.thorchain.net 
:arrow_down: 
https://thorchain.net

### Example:

https://testnet.thorchain.net/#/txs/81AF1491E3A3232C2AA921B6BCDD52E1F8AFB687044422449B8180B7C13535A8 
:arrow_down: 
https://thorchain.net/#/txs/81AF1491E3A3232C2AA921B6BCDD52E1F8AFB687044422449B8180B7C13535A8

### PR DESCRIPTION
**Pull request by veado**
- [x] Fix handling of explorer url in `PoolHistoryView.tsx` + `PoolActionsHistoryView`
- [x] Introduce `getClientNetwork` (+ few tests) 
- [x] Introduce `clientNetwork$` (to remove all duplicated `toXYZNetwork` helper function in misc. client streams)
- [x] Use latest `xchain-thorchain@0.13.3`

Close #1310

### CODE REVIEW

File: src/renderer/services/app/service.ts [hunk 1]
```diff
@@ -23,4 +25,6 @@ const { get$: getNetwork$, set: changeNetwork, get: getCurrentNetworkState } = o
 // So we do need a simple "dirty check" to provide "real" changes of selected network
 const network$ = getNetwork$.pipe(distinctUntilChanged())
 
-export { onlineStatus$, network$, changeNetwork, getCurrentNetworkState }
+const clientNetwork$ = network$.pipe(RxOp.map(getClientNetwork))
```
Comment (thatStrangeGuyThorchain) [inline src/renderer/services/app/service.ts]: just a question here.
can not we just add `RxOp.map(getClientNetwork)` to the initial `network$` stream ?

File: src/renderer/views/pool/PoolHistoryView.tsx [hunk 2]
```diff
@@ -37,10 +39,6 @@ export const PoolHistory: React.FC<Props> = ({ className, poolAsset }) => {
 
   const { getExplorerTxUrl$ } = useThorchainContext()
 
-  /**
-   * TODO @asgdxteam Remove `network` param
-   * after https://github.com/xchainjs/xchainjs-lib/issues/287 is resolved
-   */
   const { network$ } = useAppContext()
```
Comment (thatStrangeGuyThorchain) [inline src/renderer/views/pool/PoolHistoryView.tsx]: i suppose we can use directly `clientNetwork$` if provide it to the `AppContext` or with previous comment just use `network` if there is `getClientNetwork` inside of `network$`

**Review by thatStrangeGuyThorchain — DISMISSED** [review]
nice. just questions...

File: src/renderer/services/app/service.ts (hunk 1 above)
Comment (veado) [reply src/renderer/services/app/service.ts]: As discussed in DM: We will do it as soon as we know how to deal the switch `chaosnet` -> `mainnet` in the near future.

**Review by thatStrangeGuyThorchain — APPROVED** [review]

**Pull request merged**